scout/utils/ensembl_biomart_clients.py

```diff
@@ -32,3 +32,19 @@ def stream_resource(self, interval_type: str) -> Iterator[str]:
         shug_url: str = f"{SCHUG_BASE}{SCHUG_RESOURCE_URL[interval_type]}{self.build}"
 
         return self.stream_get(shug_url)
+
+    def check_integrity(self, file_path: str):
+        """Counts the number of lines containing '[success]'. One such line per chromosome is expected for the file to be considered intact."""
+
+        def count_line_occurrences(file_path, target_line):
+            with open(file_path, "r", encoding="utf-8") as file:
+                return sum(1 for line in file if line.strip() == target_line)
+
+        nr_chromosomes_in_file = count_line_occurrences(file_path, "[success]")
+        nr_chromosomes = 24
+        if nr_chromosomes_in_file < nr_chromosomes:
+            raise BufferError(
```

Comment:
This will stop the downloads (all downloads if your command is `scout download everything`), but it's fine, since the Ensembl resources are the last resources downloaded by that command, see [here](https://github.com/Clinical-Genomics/scout/blob/50de25163c351ef6cddb6d422635cd19df07ad7f/scout/commands/download/everything.py#L40)